apps/web/components/Project/Header/DefaultProjectHeader.tsx

```diff
@@ -31,45 +33,63 @@ export const DefaultProjectHeader = ({
 
   const textBackgroundColor = getTextBackgroundColor(projectPage)
 
+  const textRef = useRef<HTMLDivElement | null>(null)
+
+  const { width } = useWindowSize()
+
+  const isBelowLarge = width < theme.breakpoints.lg
+
+  const maxImageHeight = !isBelowLarge
+    ? textRef.current?.getBoundingClientRect()?.height ?? undefined
+    : undefined
+
   return (
     <Box className={defaultImageIsProvided ? styles.headerWrapper : undefined}>
-      <Box
+      <div
```

Comment:
Is there a reason for changing Box -> Div?

Reply:
Perhaps not, I've changed it back to being just a Box